src/lib.rs

```diff
@@ -13282,7 +13282,7 @@ mod tests {
       ".foo { background: linear-gradient(lch(56.208% 136.76 46.312), lch(51% 135.366 301.364)) }",
       indoc! { r#"
         .foo {
-          background: -webkit-linear-gradient(#ff0f0e, #7773ff);
+          background: -webkit-linear-gradient(top, #ff0f0e, #7773ff);
```

Comment:
Some test results include an extra `top` keyword in vendor-prefixed linear-gradient values because the original input omits the direction. When unspecified, the new syntax defaults to `to bottom`, which is converted to `top` in legacy syntax.